boto3/session.py

```diff
@@ -255,6 +255,12 @@ def client(self, service_name, region_name=None, api_version=None,
         :return: Service client instance
 
         """
+        if endpoint_url is None:
+            try:
+                endpoint_url = os.environ["AWS_ENDPOINT_URL"]
+            except KeyError:
+                pass
```

Comment:
This could do a `.get()` which returns `None` if the env var is not set.
```suggestion
            endpoint_url = os.environ.get("AWS_ENDPOINT_URL")
```

Reply:
Are we able to make this change and get the PR merged?

Reply:
Alternatively:

```
endpoint_url = endpoint_url or os.getenv('AWS_ENDPOINT_URL')
```